cli/azd/pkg/tools/azcli/spring_app.go

```diff
@@ -0,0 +1,49 @@
+package azcli
+
+import (
+	"context"
+	"fmt"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/appplatform/armappplatform"
+)
+
+type AzCliSpringAppProperties struct {
+	HostNames []string
```

Comment:
I'd rather we expose this as `Fqdn string` so there's a clearer relationship between the underlying service model type and what we return here.